awx/main/tasks/receptor.py

```diff
@@ -25,12 +25,17 @@
     cleanup_new_process,
 )
 from awx.main.constants import MAX_ISOLATED_PATH_COLON_DELIMITER
+from awx.main.models import Instance
+from awx.main.dispatch.publish import task
 
 # Receptorctl
 from receptorctl.socket_interface import ReceptorControl
 
+from filelock import FileLock
```

Comment:
maybe I'm dense, but I can't figure out where this requirement is coming from.

Reply:
can u elaborate on this comment

Reply:
We could probably use this in `acquire_lock` if it's really there.

Reply:
No, I mean, wouldn't you need to add this to the `requirements.txt`? Or is it in the standard library? Testing on python 3.10 I don't see it.

Reply:
weird... i try to run this in the development environment and it works 

well i haven't build a "normal" image yet. ur right filelock is not a standard library i added to requirements/requirements.in